### PR TITLE
feat: added 2 scenarios related to #161

### PR DESCRIPTION
**Pull request by raphaelweis**
This adds the cases mentioned in #161 to the feature.

### CODE REVIEW

**Review by benel — CHANGES_REQUESTED** [review]
Thank you very much for your interesting contribution @raphaelweis, @Barnabepetit, @louisduhalberruer, @nitram35, @petitfa1. 

1. Please follow the instructions given on Moodle:
> Dans une branche thématique (nommée selon le ticket traité, ex : "fix-999"), préparez une "contribution" (commit)  avec vos scénarios modifiés. 
> Veillez à ce que :
> - le titre soit formaté de la même manière que [les contributions analogues](https://github.com/search?q=repo%3AHypertopic%2FHyperGlosae+%5ESCENARIO+should+see&type=commits&s=committer-date&o=asc) (`SCENARIO: <nom du ticket en anglais> (see #999).`),
> - les co-auteurs apparaissent selon [la syntaxe de GitHub](https://docs.github.com/fr/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors).

2. In the discussion, please mention authors' full names  (and check with one another that you agree) on the authorization concerning all the content you will produce in this branch. 

3. Here are also several change requests.

File: features/edit_glose_content.feature [hunk 1]
```diff
@@ -36,3 +36,35 @@ Scénario: sans être connecté
...
+  Et une glose faisant référence uniquement à la partie 1
+  Quand j'essaie d'éditer le bloc 2 avec le texte 
+    """
+    Diane
+    : Acteurs
+    """
+  Alors le bloc affiche
+    """
+    Diane
+      Acteur
```
Comment (benel) [inline features/edit_glose_content.feature:53]: without `s`?!

File: features/edit_glose_content.feature [hunk 2]
```diff
@@ -36,3 +36,35 @@ Scénario: sans être connecté
...
+  Et une glose faisant référence uniquement à la partie 1
+  Quand j'essaie d'éditer le bloc 1 avec le texte 
+    """
+    Diane
+    : Acteurs
+    """
+  Alors le bloc affiche
+    """
+    Diane
+      Acteur
```
Comment (benel) [inline features/edit_glose_content.feature:69]: without `s`?!

File: features/edit_glose_content.feature [hunk 3]
```diff
@@ -36,3 +36,35 @@ Scénario: sans être connecté
...
+
+  Soit un document dont je suis l'auteur avec 2 parties
+  Et une session active avec mon compte
+  Et une glose faisant référence uniquement à la partie 1
+  Quand j'essaie d'éditer le bloc 2 avec le texte 
+    """
+    Diane
+    : Acteurs
+    """
+  Alors le bloc affiche
```
Comment (benel) [inline features/edit_glose_content.feature:50]: A clause with the user as the subject of the verb would be preferred.

File: features/edit_glose_content.feature [hunk 4]
```diff
@@ -36,3 +36,35 @@ Scénario: sans être connecté
...
+
+  Soit un document dont je suis l'auteur avec 2 parties
+  Et une session active avec mon compte
+  Et une glose faisant référence uniquement à la partie 1
+  Quand j'essaie d'éditer le bloc 1 avec le texte 
+    """
+    Diane
+    : Acteurs
+    """
+  Alors le bloc affiche
```
Comment (benel) [inline features/edit_glose_content.feature:66]: A clause with the user as the subject of the verb would be preferred.

**Review by benel — CHANGES_REQUESTED** [review]
Thank you very much for your interesting contribution @raphaelweis, @Barnabepetit, @louisduhalberruer, @nitram35, @petitfa1. 

1. Please follow the instructions given on Moodle:
> Dans une branche thématique (nommée selon le ticket traité, ex : "fix-999"), préparez une "contribution" (commit)  avec vos scénarios modifiés. 
> Veillez à ce que :
> - le titre soit formaté de la même manière que [les contributions analogues](https://github.com/search?q=repo%3AHypertopic%2FHyperGlosae+%5ESCENARIO+should+see&type=commits&s=committer-date&o=asc) (`SCENARIO: <nom du ticket en anglais> (see #999).`),
> - les co-auteurs apparaissent selon [la syntaxe de GitHub](https://docs.github.com/fr/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors).

2. In the discussion, please mention authors' full names  (and check with one another that you agree) on the authorization concerning all the content you will produce in this branch. 

3. Here are also several change requests. 

4. The Pull request should be in draft mode until a commit fix is pushed in the branch.

**Comment by raphaelweis** [comment]
We have amended the previous commit to satisfy the review.

**Comment by benel** [comment]
Thank you for your updates @raphaelweis, @Barnabepetit, @louisduhalberruer, @nitram35, @petitfa1.

- The content is great.
- The history is fine, except a little problem in the second commit: `SCENARIO` should be followed by the title of a feature or a fix being scenarized. I suggest you to rephrase the whole commit title it in something like `PROCESS: Scenario steps related to glose blocks should be reusable.`. See that as a nice opportunity for your entire team to test "interactive rebase". Of course I can help you if needed.

**Comment by benel** [comment]
Thank you for your contribution @raphaelweis, @Barnabepetit, @louisduhalberruer, @nitram35, @petitfa1.

Is it your last version?

I thought it was done but it seems that, without your fix, your test that should pass fails. Is this right?

**Comment by raphaelweis** [comment]
> Thank you for your contribution @raphaelweis, @Barnabepetit, @louisduhalberruer, @nitram35, @petitfa1.
> 
> Is it your last version?
> 
> I thought it was done but it seems that, without your fix, your test that should pass fails. Is this right?

I believe the implementation of the fix to be correct, but it broke multiple CSS selectors in the test implementations. I have looked at it a bit but so far no luck fixing the failing tests. I'll have another look at it this weekend or next week.

Once we get all the tests to pass again it should be good to merge.

**Comment by benel** [comment]
Thank you for your answer @raphaelweis.
I've found what was the problem with the tests.

On blank gloses, the testing bot – which is far faster than a normal user – "clicks" on your new block place holder before it is replaced by "&lt;TEXT&gt;".

The tests were fixed with 40c68972f27f127fcd5db4c7ca7ddd03efbad777.

Your branch is now merged. Well done  @raphaelweis, @Barnabepetit, @louisduhalberruer, @nitram35, @petitfa1!

**Pull request merged**